zerver/lib/send_email.py

```diff
@@ -50,6 +55,59 @@ def security_email_from_name(language: Optional[str]=None,
         with override_language(language):
             return _("Zulip Account Security")
 
+def send_custom_email(users: List[UserProfile], options: Dict[str, Any]) -> None:
```

Comment:
This commit is not a pure code move, based on the same `git show --color-moved-ws` approach used before; e.g. various comments are deleted.  Can you redo this commit as a pure code move (with additional commits before or after for the other changes)?

Reply:
@timabbott  I've splitted this commit into two (so one edit some stuff, and the other one only move code and adjust imports).